cyberark/conjur
# Binpossum has been renamed to binconjurctl  183

Closes #183

#### What does this pull request do?

Renames `bin/possum` to `bin/conjurctl`

#### Where should the reviewer start?

By examining the file changes in `bin/`

#### How should this be manually tested?

Run `build.sh`, then run `dev/start.sh` and confirm that the possum container server is now started with the command `conjurctl server`. You can also run `test.sh` and `dev/cucumber.sh` if you like.

#### Link to build in Jenkins

https://jenkins.conjur.net/view/V5%20Possum%20Jobs/job/possum/job/binpossum-has-been-renamed-to-binconjurd--183/
